osh/hub/osh_xmlrpc/scan.py

```diff
@@ -237,30 +237,36 @@ def find_tasks(request, query):
      * 'nvr': search by specific NVR
      * 'regex': find by provided regex (this is not match, but find, if you
                 want match, change your regex to "^<regex>$")
+     * 'comment': string, search by comment
+
+    Query also supports following optional keys:
+     * 'states': list, search by task states
 
     Returned list is ordered by date, when task finished -- latest task is
     first. Unfinished tasks are at the tail. If there is any problem with
     querying, empty list is returned.
     """
     if not isinstance(query, dict):
         return []
-    nvr = query.get('nvr')
     package_name = query.get('package_name')
-    comment = query.get('comment')
+    nvr = query.get('nvr')
     regex = query.get('regex')
+    comment = query.get('comment')
     states = query.get('states')
     latest = query.get('latest')
 
     result = []
-    tasks = None
     if nvr:
```

Comment:
Off-topic:  This PR made me realise that the searching is broken/unintuitive when multiple options are specified, e.g.:
```bash
$ osh-cli find-tasks --nvr --comment test
```
is accepted but the comment in this function says that:

> Query is a dict (hash/map), it has to have exactly one of these keys:

This is not checked anywhere so CLI will still return some results...

@kdudka We should definitely fix this!

Reply:
filed an issue so we get somewhere to track: https://github.com/openscanhub/openscanhub/issues/268

Reply:
Thank you!